src/tcgwars/logic/impl/gen8/BlueSkyStream.groovy

```diff
@@ -548,15 +574,20 @@ public enum BlueSkyStream implements LogicCardInfo {
         move "Powder Snow", {
           text "30 damage. Your opponent's Active Pokémon is now Asleep."
           energyCost W, C
-          attackRequirement {}
           onAttack {
             damage 30
+            applyAfterDamage ASLEEP
           }
         }
         move "Daruma Headbutt", {
           text "130 damage. If this Pokemon has any damage counters on it, this attack can be used for [W]."
-          energyCost W, W, C
-          attackRequirement {}
+          energyCost W
```

Comment:
I think this should be done as a custom ability that modifies the attack cost. The current implementation breaks with effects that increase attack cost. Ex. If this Pokémon has WWC attached and no damage counters, and there is a wondrous labyrinth in play, it could still attack.

Reply:
This is copied from Boltund BST. While I agree that if it is causing bugs it should be fixed, I don't think it should delay the set if another set already does it this way. It can be fixed for both at the same time (and I don't really have time to figure it out atm)

Reply:
I think Boltund copied it from tapu fini? I agree that this seems like enough of an edge case that we can leave it for now.